### PR TITLE
Add offline error to the send page

### PR DESCRIPTION
**Pull request by GBKS**
As requested in #673.

The send page has a paragraph that outlines how to about about handling errors, which includes a few example. I added this "offline error" to that slider.

Also added a table of contents at the top of the page, to more easily see what's covered.

🔗[Check the preview](https://deploy-preview-680--sad-borg-390916.netlify.app/guide/payments/send/#errors)🖇

### CODE REVIEW

File: guide/payments/send.md [hunk 1]
```diff
@@ -77,6 +77,9 @@ imagesErrors:
     - file: generic-error-details
       alt: A screen showing error details
       caption: Access to error details for problem-solving.
+    - file: offline-error
+      alt: A screen showing error details
```
Comment (Bosch-0) [inline guide/payments/send.md:81]: How would the send message button work?

Comment (GBKS) [reply guide/payments/send.md:81]: "Send message" would open the native share sheet with a pre-written message like "I want to send you some bitcoin. Please open your wallet application to receive them." The user could just pick the recipient and send it off.

**Review by sbddesign — APPROVED** [review]
LGTM

File: guide/payments/send.md (hunk 1 above)
Comment (Bosch-0) [reply guide/payments/send.md:81]: Makes sense :)

**Pull request merged**